### PR TITLE
where patch

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-740.mdx [hunk 1]
```diff
@@ -18,15 +22,25 @@ const int MAX_N = 20;
...
+	int i2, j2;
+
+	bool is_inside(PCL o) {
+		return (
+			i1 >= o.i1 &&
+			i2 <= o.i2 &&
+			j1 >= o.j1 &&
+			j2 <= o.j2
+		);
+	}
```
Comment (ryanchou-dev) [inline solutions/usaco-740.mdx:37]: better name than o?

File: solutions/usaco-740.mdx [hunk 2]
```diff
@@ -57,21 +71,21 @@ bool is_pcl(int i1, int j1, int i2, int j2) {
...
 	// Reset visited vector for the next call
-	fill(visited.begin(), visited.end(), vector<bool>(MAX_N, false));
+	fill(visited.begin(), visited.end(), vector<bool>(MAX_N));
 
 	// Verify PCL conditions
 	int color_count = 0;
 	bool color_with_one_region = false;
 	bool color_with_more_regions = false;
-	for (int i = 0; i < 26; i++) {
+	for (int i = 0; i < region_count.size(); i++) {
```
Comment (ryanchou-dev) [inline solutions/usaco-740.mdx:88]: (int)(region_count.size())

File: solutions/usaco-740.mdx [hunk 3]
```diff
@@ -90,233 +104,197 @@ bool is_pcl(int i1, int j1, int i2, int j2) {
...
-				throw new RuntimeException(e);
-			}
-			return str;
+		public boolean isInside(PCL b) {
+			return (
+				i1 >= b.i1 &&
+				i2 <= b.i2 &&
+				j1 >= b.j1 &&
+				j2 <= b.j2
+			);
```
Comment (ryanchou-dev) [inline solutions/usaco-740.mdx:297]: sabne

**Review by ryanchou-dev — APPROVED** [review]

File: solutions/usaco-740.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-740.mdx:37]: i mean it's `o` for `other`, should i rename it to that?

File: solutions/usaco-740.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/usaco-740.mdx:88]: why?

File: solutions/usaco-740.mdx (hunk 1 above)
Comment (envyaims) [reply solutions/usaco-740.mdx:37]: ya

**Review by envyaims — APPROVED** [review]
ayo

**Pull request merged**